dronecode_sdk/__init__.py

```diff
@@ -51,3 +51,33 @@ def connect(*args, **kwargs):
         globals()[plugin](plugin_manager)
 
     return plugin_manager
+
+
+import atexit
+import os
+import subprocess
+import sys
+
+from dronecode_sdk import bin
+from time import sleep
+
+if sys.version_info >= (3,7):
+    from importlib.resources import path
+else:
+    from importlib_resources import path
+
+def start_mavlink():
```

Comment:
This should work, but somehow we have to make sure we kill the process when python exits (due to a failure or sth else). It _should_ but it happens sometimes that it can't manage to cancel it

Reply:
You mean `atexit` will not always kill it? I believe there is no way to make sure of that. It seems to work well when the program stops normally or raises an error. If you SIGKILL, then there is no way (the Python code cannot intercept that, otherwise it could ignore it and that's not the point of SIGKILL).

I can't find a better way to run the server in its own process without ever having a chance to leak. We could run it as a thread, but that requires to wrap `libbackend.so` (maybe with Cython?).

Reply:
That's correct! I was thinking about a randomized port for the GRPC service.

Cython might get more complicated, I'd suggest to stick with as few dependencies as possible!

Reply:
Good point for the randomized port. Let's keep that in mind as an enhancement for now. But I agree it should be done!